src/uu/ps/src/ps.rs

```diff
@@ -0,0 +1,99 @@
+// This file is part of the uutils procps package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+#[cfg(target_os = "linux")]
+use clap::crate_version;
+use clap::{Arg, ArgAction, Command};
+use uucore::{error::UResult, format_usage, help_about, help_usage};
+
+const ABOUT: &str = help_about!("ps.md");
+const USAGE: &str = help_usage!("ps.md");
+
+#[uucore::main]
+pub fn uumain(args: impl uucore::Args) -> UResult<()> {
+    let matches = uu_app().try_get_matches_from(args)?;
+
+    Ok(())
+}
+
+pub fn uu_app() -> Command {
+    Command::new(uucore::util_name())
+        .version(crate_version!())
+        .about(ABOUT)
+        .override_usage(format_usage(USAGE))
+        .infer_long_args(true)
+        .disable_help_flag(true)
+        .arg(Arg::new("help").long("help").action(ArgAction::Help))
+        .args([
+            Arg::new("A")
+                .short('A')
+                .help("all processes")
+                .visible_alias("e")
+                .action(ArgAction::SetTrue),
+            Arg::new("a")
+                .short('a')
+                .help("all with tty, except session leaders")
+                .action(ArgAction::SetTrue),
+            // Arg::new("a_")
+            //     .short('a')
+            //     .help("all with tty, including other users")
+            //     .action(ArgAction::SetTrue)
+            //     .allow_hyphen_values(true),
```

Comment:
Regarding this part of the code, I want to implement this part of the ps command, but clap doesn't allow me to do so, what should I do?

```shell
Basic options:
 -A, -e               all processes
 -a                   all with tty, except session leaders
  a                   all with tty, including other users
 -d                   all except session leaders
 -N, --deselect       negate selection
  r                   only running processes
  T                   all processes on this terminal
  x                   processes without controlling ttys
```

Reply:
Currently I don't know how we should solve it :| Maybe we have to do it manually, with a custom parser.

Reply:
Sounds like bad news :( , is there an example of this in uutils?

Reply:
It's simply because `ps` is complex and supports three kinds of options: unix options with one dash, BSD options without a dash, and GNU long options with two dashes.

`dd` in coreutils uses its own parser.

Reply:
I'll complet the GNU options and *NIX options in this pr, and add custom parser in other pr